packages/block-editor/src/components/use-resize-canvas/index.js

```diff
@@ -11,11 +11,15 @@ import { default as useSimulatedMediaQuery } from '../../components/use-simulate
 /**
  * Function to resize the editor window.
  *
- * @param {string} deviceType Used for determining the size of the container (e.g. Desktop, Tablet, Mobile)
+ * @param {string}  deviceType                  Used for determining the size of the container (e.g. Desktop, Tablet, Mobile)
+ * @param {boolean} __unstableDisableSimulation Whether to disable media query simulation.
  *
  * @return {Object} Inline styles to be added to resizable container.
  */
-export default function useResizeCanvas( deviceType ) {
+export default function useResizeCanvas(
+	deviceType,
+	__unstableDisableSimulation
```

Comment:
I  think at some point, when the iframe becomes more the "default" we should consider removing this entirely. (no responsive support for non-iframes)

Reply:
Agreed, but for now it's still used by the post editor.